README.md

```diff
@@ -137,7 +137,7 @@ Once you have earned the Responsive Web Design, Algorithms and Data Structures,
 
 #### Legacy Certifications
 
-We also have 4 legacy certifications dating back to our 2015 curriculum, which are still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
+We also have 4 legacy certifications dating back to our 2015 curriculum, which is still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
```

Comment:
Here, I believe `are` refers to the "4 legacy certifications", which is the correct usage.